server/lib/auto_rotated_keys.js

```diff
@@ -1,8 +1,19 @@
-const { cookieMaxAge, autoRotateKeys: leadingServer } = require('config')
-const __ = require('config').universalPath
+// This module recovers keys in config/.sessions_keys, and creates and keeps updated
+// an array of keys that will be passed to auth middlewares to sign cookies
+
+// There are presently no automated tests for this behavior, but manual tests can be run
```

Comment:
maybe some tests could be derived from `tests/api/auth/logout` ?
also the aforementioned test is red on my setup (and green on `master`)

Reply:
* logout test fixed in 7ac297e
* added some tests on sessions in e26dace